templates/workloads/partials/cf/env-controller.yml

```diff
@@ -1,15 +1,10 @@
 EnvControllerAction:
-  Metadata:
-    'aws:copilot:description': "Update your environment's shared resources"
-  Type: Custom::EnvControllerFunction
-  Properties:
-    ServiceToken: !GetAtt EnvControllerFunction.Arn
-    Workload: !Ref WorkloadName
-    EnvStack: !Sub '${AppName}-${EnvName}'
-    Parameters: 
-      - 'ALBWorkloads'
-    # We need to force trigger this lambda function on all deployments, so we give it a random ID as input on all event types.
-    UpdateID: {{ randomUUID }}
+{{- if eq .WorkloadType "Load Balanced Web Service"}}
+{{include "env-controller-action-with-alb" . | indent 2}}
+{{- end}}
+{{- if eq .WorkloadType "Backend Service"}}
```

Comment:
```suggestion
{{- else }}
```
We also have `network` enabled for Jobs (https://aws.github.io/copilot-cli/docs/manifest/scheduled-job/#network) can we include this template for jobz as well?

Reply:
Solved by making this logic to a function.